src/DonationForms/resources/styles/components/_gateways.scss

```diff
@@ -1,192 +1,96 @@
 .givewp-fields-gateways {
-
-    .givewp-test-mode-notice {
+    &__list {
         display: flex;
-        justify-content: left;
-        align-items: center;
-        width: 100%;
-        padding: 0.75rem 1rem;
-        gap: 1rem;
-        margin-bottom: 2.5rem;
-        border-radius: 0.25rem;
-        border-top: 1px solid var(--orange-400, #F29718);
-        border-right: 1px solid var(--orange-400, #F29718);
-        border-bottom: 1px solid var(--orange-400, #F29718);
-        border-left: 4px solid var(--orange-400, #F29718);
-        background: var(--shades-white, #FFF);
-
-        svg {
-            width: 1.5rem;
-            height: 1.5rem;
-            flex-shrink: 0;
-        }
-
-        p {
-            margin: 0;
-            line-height: 1.25rem;
-            color: var(--grey-900, #0E0E0E);
-            font-size: 0.875rem;
-            font-style: normal;
-            font-weight: 400;
-        }
+        flex-direction: column;
+        gap: var(--givewp-spacing-2);
+        list-style: none;
     }
 
-    ul {
+    &__gateway {
         display: flex;
         flex-direction: column;
-        gap: 0.9375rem;
-        list-style: none;
-    }
+        gap: var(--givewp-spacing-1);
 
-    li {
-        --tab-radio-diameter: 1.25rem;
-        --tab-radio-border-size: 0.0625rem;
-        --tab-block-padding: 1.375rem;
-        --tab-block-size: calc(var(--tab-block-padding) * 2 + var(--tab-radio-diameter));
-        --tab-inline-padding: 1.375rem;
-        --tab-transition: 0.2s ease-in-out;
-        position: relative;
-        -webkit-padding-before: var(--tab-block-size);
-        padding-block-start: var(--tab-block-size);
-        background-color: #fafafa;
+        background-color: var(--givewp-shades-white);
         transition: background-color var(--tab-transition);
```

Comment:
idk if this is the one causing it, but the timing on the gateway radio color transition feels too slow in relation to the accordion style dropdown being immediate.  The dropdown opens and then the radio color follows.  I know its subjective but it bothers me lol

Reply:
I'm not sure I follow. Hahah! Are the fields themselves supposed to transition in? May need your help on this one.